src/main/java/com/redhat/devtools/intellij/tektoncd/actions/logs/LogsBaseAction.java

```diff
@@ -122,17 +127,15 @@ private String pickRunWithDialogHelper(List resourceRunsName, String kind, Strin
 
     }
 
-    private String getNamespace(LazyMutableTreeNode selected) {
-        Class<?> nodeClass = selected.getClass();
-        if (PipelineNode.class.equals(nodeClass) || TaskNode.class.equals(nodeClass)) {
-            return selected.getParent().getParent().toString();
-        } else {
-            if (((RunNode) selected).getLevel() == 4) {
-                return selected.getParent().getParent().getParent().toString();
+    private String getNamespace(Object selected) {
```

Comment:
Is it possible to use root.client.namespace to retrieve the current element namespace? I'm asking because we will have a similar thing with taskrun/pipelinerun when adding them as root nodes